test/browser/integration/classes.js

```diff
@@ -27,7 +27,7 @@ describe('classes', function() {
               name = name.replace('-', '');
               expect(Modernizr[name]).to.be(result);
             } else {
-              name = name.split('-');
+              name = name.split(/-(.+)/); // split at first occurence of '-'
```

Comment:
`inputtypes` has a `datettime-local` property. Without this change, we would split the name `inputtypes-datettime-local` into three instead of two parts.